public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -333,6 +333,9 @@ define(function(require) {
       $("#navbar-publish-button").click(showPublishDialog);
       $("#publish-button-cancel").click(hidePublishDialog);
 
+      //Publish link
+      $("#link-publish-link").click(hidePublishDialog);
```

Comment:
does this trigger when right-clicking to copy the URL? If so this might need a bit of filter code so that the hide action only kicks in for "left button" clicks